Modules/Registration/PDEDeformable/test/itkMultiResolutionPDEDeformableRegistrationTest.cxx

```diff
@@ -244,9 +244,8 @@ itkMultiResolutionPDEDeformableRegistrationTest(int argc, char * argv[])
   registrator->GetModifiableFixedImagePyramid()->UseShrinkImageFilterOn();
   registrator->GetModifiableMovingImagePyramid()->UseShrinkImageFilterOn();
 
-  unsigned int     numLevel = 3;
-  constexpr size_t arraySize = 10;
```

Comment:
I think this test failed sporadically in nightly dashboard testing. Possible reason for not segfaulting during testing: compiler optimizes away undefined behavior in release mode.

Reply:
Thanks for the review @dzenanz. I had a very vague memory of seeing this test segfault in the dashboard, but I was not able to find an example yesterday, and hence my comment.